SITnett/templates/registration/login.html

```diff
@@ -0,0 +1,24 @@
+{% extends "meny.html" %}
+
+{% block innhold %}
+
+  {% if next %}
+    {% if user.is_authenticated %}
+    Kontoen din har ikke tilgang til denne siden! Logg inn med en gyldig konto.<br>
```

Comment:
Det kan jo hende de ikke _skal_ ha tilgang til denne siden. Kan vurdere å endre meldingen til et enkelt `Kontoen din har ikke tilgang til denne siden.` Stemmer også for å droppe utropstegn til fordel for punktum.

Reply:
Enig.